jlm/llvm/opt/alias-analyses/Andersen.cpp

```diff
@@ -1003,4 +1006,12 @@ Andersen::ConstructPointsToGraphFromPointerObjectSet(
   return pointsToGraph;
 }
 
+std::unique_ptr<PointsToGraph>
+Andersen::ConstructPointsToGraphFromPointerObjectSet(const PointerObjectSet & set)
+{
+  // Create a throwaway instance of statistics
+  auto statistics = Statistics::Create(jlm::util::filepath(""));
```

Comment:
There is no need to create an instance on the heap here. It could be on the stack, no?